src/js/components/DeploymentsListComponent.jsx

```diff
@@ -107,6 +168,11 @@ var DeploymentListComponent = React.createClass({
               Error fetching deployments. Refresh to try again.
             </td>
           </tr>
+          <tr className={errorMessageClassSet}>
+            <td className="text-center text-danger" colSpan="5">
+              {state.errorMessage}
+            </td>
+          </tr>
```

Comment:
Maybe we should have this below the "no deployments" row to align with apps:
![](http://cl.ly/image/3J352C1f1m2L/Image%202015-06-17%20at%2012%3A02%3A28.png)
![](http://cl.ly/image/1E200N2E1R1e/Image%202015-06-17%20at%2012%3A04%3A15.png)